tests/compliance/json/conftest.py

```diff
@@ -0,0 +1,211 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import json
+
+import numpy as np
+import pandas as pd
+import pandas._testing as tm
+import pytest
+
+from db_dtypes import JSONArray, JSONDtype
+
+
+def make_data():
+    # Sample data with varied lengths.
+    samples = [
+        {"id": 1, "bool_value": True},  # Boolean
```

Comment:
Could we mix in some rows that are just None/null, boolean, float, int, string, list/array please? In BigQuery JSON type can store any valid JSON object not just dictionaries.

Reply:
Good points! Added all other types. The `None` has been added into `data_missing`,

Reply:
Done.